fbrp/src/fbrp/runtime/conda.py

```diff
@@ -148,20 +150,19 @@ async def death_handler(self):
 
     async def handle_down(self):
         try:
-            if self.proc.returncode is not None:
-                return
-            proc_pid = self.get_pid()
+            if self.proc.returncode is None:
+                proc_pid = self.get_pid()
```

Comment:
why not use `self.proc.pid`

Reply:
Same thing. Some external utilities need access to the pid regardless of runtime environment (docker, conda, ...), so `get_pid()` is a common point for this.